packages/vega-interpreter/src/functions.js

```diff
@@ -75,6 +75,11 @@ export default {
   split:        function() { return apply('split', arguments, String); },
   replace:      function() { return apply('replace', arguments, String); },
   trim:         x => String(x).trim(),
+  // Base64 encode/decode
+  // Convert binary string to base64-encoded ascii
+  btoa:         x => btoa(x),
```

Comment:
Could this just be

```suggestion
  btoa:         btoa,
```

Reply:
It could. A small difference is wrapping btoa in a function ensures that our public API ensures that only 1 argument is ever passed to `btoa`. 

This is similar to how writing `['1','2'].map(parseInt)` and `['1','2'].map((num) => parseInt(num))` are not equivalent

```js
['5','2'].map((num) => parseInt(num))
// returns [5, 2]
['5','2'].map(parseInt)
// returns [5, NaN]
```

The NaN appears as a result of calling `parseInt(2, 1)` since the index of the array is passed as the second arg of the mapping function, and changes the radix to something other than base 10. We want to avoid that.